src/ape/pytest/fixtures.py

```diff
@@ -15,14 +15,27 @@ def __init__(self):
 
     @pytest.fixture(scope="session")
     def accounts(self) -> List[TestAccountAPI]:
+        """
+        The ape account test manager containing pre-funded test accounts.
```

Comment:
NOTE: When you do `pytest -q --fixtures` with ape installed, you get special docs for each fixture built from these doc strs.

So I want to avoid putting links in these.

Before, it was read and bad looking because they are missing. Now, it looks more profess

Reply:
Can we add a comment saying that we don't want links